community/transmission-daemon/build

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sbindir=/usr/bin \
+    --mandir=/usr/share/man \
+    --sysconfdir=/etc \
+    --localstatedir=/var \
+    --disable-nls \
+    --enable-cli \
```

Comment:
Does this install with `make -C daemon intall`?

Reply:
removed. `cli` is different from daemon/remote.